src/test/java/com/wikia/webdriver/testcases/desktop/adstests/TestAdsRepeatableIncontentBoxadMobileWiki.java

```diff
@@ -53,27 +53,37 @@ private static String urlWithInstantGlobals(boolean enabled) {
   public void adDisplayedBeforeSections() {
     AdsBaseObject ads = new AdsBaseObject();
     ads.getUrl(urlWithInstantGlobals(true));
+    int headerOffset = 0;
 
+    if (ads.hasTopBoxad()) {
+      headerOffset = 1;
+    }
 
-    ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(1))));
+    ads.scrollTo(By.id(String.format(ARTICLE_HEADER_SELECTOR, Integer.toString(headerOffset + 1))));
     Assertion.assertTrue(isIncontenAdDisplayed(1, ads),
-                         "IncontentBoxad is not displayed before section"
+            "IncontentBoxad is not displayed before section"
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "IncontentBoxad is not displayed before section" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)